configs/projects/puppet-agent.rb

```diff
@@ -1,34 +1,63 @@
 project "puppet-agent" do |proj|
-  # Project level settings our components will care about
-  proj.setting(:install_root, "/opt/puppetlabs")
-  proj.setting(:prefix, File.join(proj.install_root, "puppet"))
+  platform = proj.get_platform
 
-  if platform.is_eos?
-    proj.setting(:sysconfdir, "/persist/sys/etc/puppetlabs")
-    proj.setting(:link_sysconfdir, "/etc/puppetlabs")
-  elsif platform.is_osx?
-    proj.setting(:sysconfdir, "/private/etc/puppetlabs")
+  # Project level settings our components will care about
+  # Windows has its own seperate layout
+  if platform.is_windows?
+    proj.setting(:install_root, "#{platform.drive_root}/ProgramFiles/PuppetLabs")
```

Comment:
Also I should mention that the above isn't 100% reliable, as it could be Progra~2 or Puppet~2. It all depends on what other directories exist. That said if we're building on clean VMs I'm less concerned.